deep_quoridor/src/train_sb3.py

```diff
@@ -121,6 +122,11 @@ def mask_fn(env):
     return env.action_mask()
 
 
+def make_model_id(env_kwargs):
+    # HACK: This is takenfrom trainable_agent model_id()
+    return f"sb3ppo_B{env_kwargs['board_size']}W{env_kwargs['max_walls']}_mv0"
```

Comment:
If I understand correctly this is because we don't have an instance of the model here, no? my worry is that it may get out of sync (e.g. you hardcoded the model version to 0 here).  Maybe we can just instantiate the model class without a filename and wandb so that nothing is loaded (should be pretty light) just to get the model id?
Otherwise don't worry, we can leave it like this

Reply:
ok, I went ahead and started using an instance to get the name, so that it's all in a single place and the verison name is not hardcoded.
I had to move more stuff around than I wanted to do it, but I think it's a small step in the right direction.